### PR TITLE
more fancy homepage

### PR DESCRIPTION
**Pull request by szmo**
https://trello.com/c/ki2sldcj/31-homepage-dodac-content

### CODE REVIEW

File: rails_app/app/views/homepage/index.html.haml [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
-    %input.form-control{type: 'text', placeholder: 'www.example.com'}
-    %span.input-group-btn
-      %button.btn.btn-default{type: 'button', type: 'button'}
-        Search!
+  - if current_user
+    = image_tag 'phone1.gif'
+    .header
+      %h1 Paster url to track!
+    .input-group.search
+      %input.form-control{type: 'text', placeholder: 'www.example.com'}
```
Comment (evemorgen) [inline rails_app/app/views/homepage/index.html.haml]: I would rather have some supported site, like `itaka.pl` or `x-kom.pl`

File: rails_app/app/views/homepage/index.html.haml [hunk 2]
```diff
@@ -1,6 +1,32 @@
 .well
-  .input-group
-    %input.form-control{type: 'text', placeholder: 'www.example.com'}
-    %span.input-group-btn
-      %button.btn.btn-default{type: 'button', type: 'button'}
-        Search!
+  - if current_user
+    = image_tag 'phone1.gif'
+    .header
+      %h1 Paster url to track!
```
Comment (evemorgen) [inline rails_app/app/views/homepage/index.html.haml]: `Paste` if you will, but I think this whole `h1` is unnecessary

File: rails_app/app/views/homepage/index.html.haml [hunk 3]
```diff
@@ -1,6 +1,32 @@
...
+          Search!
+  - else
+    .header
+      %h1 Welcome to PRICE MONITOR as a service
+    .banner
+      = image_tag 'logos1.gif'
+    .section
+      %h3 Please sign up and don't waste Your time for tracking price. Let us do it for You!
+      .privilege
+        %h2 When You are logged in You can:
```
Comment (evemorgen) [inline rails_app/app/views/homepage/index.html.haml]: more like 
> Joining price monitor lets you:
> - track your desired DEAL
> - receive ALERTS about price change
> - VISUALIZE price change. Easily and Beautifully.

File: rails_app/app/views/homepage/index.html.haml [hunk 4]
```diff
@@ -1,6 +1,32 @@
...
+        %ul
+          .list-items
+            %li
+              %h4 add deals to track
+            %li
+              %h4 being notified anytime price changes
+            %li
+              %h4 watch price metrics for each of Your deal
+      %h2 For any further information feel free to contact us!
```
Comment (evemorgen) [inline rails_app/app/views/homepage/index.html.haml]: > Feel like talking? Contact us!

File: rails_app/app/views/homepage/index.html.haml [hunk 5]
```diff
@@ -1,6 +1,32 @@
...
+      %span.input-group-btn
+        %button.btn.btn-default{type: 'button', type: 'button'}
+          Search!
+  - else
+    .header
+      %h1 Welcome to PRICE MONITOR as a service
+    .banner
+      = image_tag 'logos1.gif'
+    .section
+      %h3 Please sign up and don't waste Your time for tracking price. Let us do it for You!
```
Comment (evemorgen) [inline rails_app/app/views/homepage/index.html.haml]: > Don't waste any of Your precious time for checking price again, let us do it for You!

File: rails_app/app/views/homepage/index.html.haml [hunk 6]
```diff
@@ -1,6 +1,32 @@
...
+    .header
+      %h1 Paster url to track!
+    .input-group.search
+      %input.form-control{type: 'text', placeholder: 'www.example.com'}
+      %span.input-group-btn
+        %button.btn.btn-default{type: 'button', type: 'button'}
+          Search!
+  - else
+    .header
+      %h1 Welcome to PRICE MONITOR as a service
```
Comment (evemorgen) [inline rails_app/app/views/homepage/index.html.haml]: I think that this `h1` should be split to 3 separate lines, like
```
Welcome to
PRICE MONITOR
as a service
```

**Comment by evemorgen** [comment]
registration is no longer working :(
screen shot 2017-11-15 at 15 38 09

File: rails_app/app/views/homepage/index.html.haml (hunk 2 above)
Comment (szmo) [reply rails_app/app/views/homepage/index.html.haml]: h1 is styling, .header is wraping

**Review by evemorgen — DISMISSED** [review]

**Pull request merged**